addons/web/static/src/js/views/kanban/kanban_model.js

```diff
@@ -286,13 +317,36 @@ var KanbanModel = BasicModel.extend({
      */
     save: function (recordID) {
         var def = this._super.apply(this, arguments);
+        const element = this.localData[recordID];
+        if (element.type === 'list' && element.activeFilter) {
```

Comment:
This check is duplicated, should we move it to `_reloadProgressBarGroupFromRecord`?